src/NzbDrone.Common.Test/Http/HttpUriFixture.cs

```diff
@@ -0,0 +1,84 @@
+﻿using FluentAssertions;
+using NUnit.Framework;
+using NzbDrone.Common.Http;
+using NzbDrone.Test.Common;
+
+namespace NzbDrone.Common.Test.Http
+{
+    public class HttpUriFixture : TestBase
+    {
+        [TestCase("", "", "")]
+        [TestCase("/", "", "/")]
+        [TestCase("base", "", "base")]
+        [TestCase("/base", "", "/base")]
+        [TestCase("/base/", "", "/base/")]
+        [TestCase("", "relative", "relative")]
+        [TestCase("", "/relative", "/relative")]
+        [TestCase("/", "relative", "/relative")]
+        [TestCase("/", "/relative", "/relative")]
+        [TestCase("base", "relative", "relative")]
+        [TestCase("base", "/relative", "/relative")]
+        [TestCase("/base", "relative", "/relative")]
```

Comment:
No, coz for Urlbase in the indexers we use CombinePath (the second set of tests), which is now simplified to always insert a slash were needed.

The top part (uri combine) is only used in the operator + (for example [here](https://github.com/Sonarr/Sonarr/blob/2fabe2d198bcba79439f22e6d1a387b76b88b722/src/NzbDrone.Core/Indexers/RssParser.cs#L270))
And uses all the logic to handle all those different relative uri formats.